pkg/plugins/policies/meshratelimit/api/v1alpha1/meshratelimit.go

```diff
@@ -90,14 +68,14 @@ type HeaderValue struct {
 type LocalTCP struct {
 	// Define if rate limiting should be disabled.
 	// Default: false
-	// +optional
-	Disabled bool `json:"disabled,omitempty"`
-
-	// The number of connections that RateLimiter allows
-	// +required
-	Connections uint32 `json:"connections,omitempty"`
+	Disabled *bool `json:"disabled,omitempty"`
+	// Defines how many connections are allowed per interval.
+	ConnectionRate *Rate `json:"connectionRate,omitempty"`
+}
 
+type Rate struct {
+	// The number of tokens that are added every interval.
```

Comment:
Since this `Rate` struct is now used both for `http.RequestRate` and `tcp.ConnectionRate` it became pretty generic. Comments for `http.RequestRate` and `tcp.ConnectionRate` are more helpful. But I rephrased `Rate` comments as well to not use `token` related words in it.